### PR TITLE
feat(translation): added full Hebrew translation

### PR DESCRIPTION
**Pull request by mobihen**
#### Description
Added Hebrew translation to the project. 
I ran it and translated everything live.

### CODE REVIEW

**Comment by mobihen** [comment]
@Fallenbagel 
Hi :) 
Can you please review?

**Review by Fallenbagel — APPROVED** [review]
LGTM!

File: src/i18n/locale/he.json [hunk 1]
```diff
@@ -1,175 +1,1324 @@
...
+  "components.Discover.FilterSlideover.ratingText": "Ratings between {minValue} and {maxValue}",
+  "components.Discover.FilterSlideover.releaseDate": "Release Date",
+  "components.Discover.FilterSlideover.runtime": "Runtime",
+  "components.Discover.FilterSlideover.runtimeText": "{minValue}-{maxValue} minute runtime",
+  "components.Discover.FilterSlideover.streamingservices": "Streaming Services",
+  "components.Discover.FilterSlideover.studio": "Studio",
+  "components.Discover.FilterSlideover.tmdbuserscore": "TMDB User Score",
+  "components.Discover.FilterSlideover.tmdbuservotecount": "TMDB User Vote Count",
+  "components.Discover.FilterSlideover.to": "To",
+  "components.Discover.FilterSlideover.voteCount": "Number of votes between {minValue} and {maxValue}",
```
Comment (gauthier-th) [inline src/i18n/locale/he.json]: here

File: src/i18n/locale/he.json [hunk 2]
```diff
@@ -1,175 +1,1324 @@
...
+  "components.Discover.MovieGenreList.moviegenres": "סרטים לפי ג׳אנר",
+  "components.Discover.MovieGenreSlider.moviegenres": "ג׳אנר סרטים",
+  "components.Discover.NetworkSlider.networks": "רשתות",
+  "components.Discover.PlexWatchlistSlider.emptywatchlist": "Media added to your <PlexWatchlistSupportLink>Plex Watchlist</PlexWatchlistSupportLink> will appear here.",
```
Comment (gauthier-th) [inline src/i18n/locale/he.json]: here

File: src/i18n/locale/he.json [hunk 3]
```diff
@@ -1,175 +1,1324 @@
...
-  "components.Discover.TvGenreSlider.tvgenres": "סוגי סדרות",
-  "components.Discover.recentlyAdded": "נוספו לאחרונה",
-  "components.Discover.recentrequests": "בקשות אחרונות",
-  "components.Discover.trending": "חמים",
+  "components.Discover.TvGenreList.seriesgenres": "ג׳אנר סדרות",
+  "components.Discover.TvGenreSlider.tvgenres": "ג׳אנרים סדרות",
+  "components.Discover.createnewslider": "יצירת סליידר",
+  "components.Discover.customizediscover": "Customize Discover",
+  "components.Discover.discover": "Discover",
+  "components.Discover.emptywatchlist": "Media added to your <PlexWatchlistSupportLink>Plex Watchlist</PlexWatchlistSupportLink> will appear here.",
```
Comment (gauthier-th) [inline src/i18n/locale/he.json]: here

File: src/i18n/locale/he.json [hunk 4]
```diff
@@ -1,175 +1,1324 @@
...
+  "components.Discover.networks": "רשתות",
+  "components.Discover.plexwatchlist": "רשימת הצפייה שלך",
+  "components.Discover.popularmovies": "סרטים פופולריים",
+  "components.Discover.populartv": "הסדרות הפופולריות",
+  "components.Discover.recentlyAdded": "התווסף לאחרונה",
+  "components.Discover.recentrequests": "התבקשו לאחרונה",
+  "components.Discover.resetfailed": "Something went wrong resetting the discover customization settings.",
+  "components.Discover.resetsuccess": "Sucessfully reset discover customization settings.",
+  "components.Discover.resettodefault": "איפוס לברירת מחדל",
+  "components.Discover.resetwarning": "Reset all sliders to default. This will also delete any custom sliders!",
```
Comment (gauthier-th) [inline src/i18n/locale/he.json]: here too

File: src/i18n/locale/he.json [hunk 5]
```diff
@@ -1,175 +1,1324 @@
...
+  "components.DownloadBlock.estimatedtime": "{time} בערך",
+  "components.DownloadBlock.formattedTitle": "{title}: עונה {seasonNumber} פרק {episodeNumber}",
+  "components.IssueDetails.IssueComment.areyousuredelete": "?האם למחוק את התגובה",
   "components.IssueDetails.IssueComment.delete": "מחיקת תגובה",
-  "components.IssueDetails.IssueComment.areyousuredelete": "האם תרצה למחוק את התגובה?",
-  "components.IssueDetails.IssueComment.edit": "לערוך תגובה",
-  "components.IssueDetails.IssueDescription.edit": "ערוך תיאור",
+  "components.IssueDetails.IssueComment.edit": "עריכת תגובה",
+  "components.IssueDetails.IssueComment.postedby": "Posted {relativeTime} by {username}",
+  "components.IssueDetails.IssueComment.postedbyedited": "Posted {relativeTime} by {username} (Edited)",
```
Comment (gauthier-th) [inline src/i18n/locale/he.json]: here too

File: src/i18n/locale/he.json [hunk 6]
```diff
@@ -1,175 +1,1324 @@
...
+  "components.IssueModal.issueOther": "אחר",
+  "components.IssueModal.issueSubtitles": "כתוביות",
+  "components.IssueModal.issueVideo": "וידאו",
+  "components.LanguageSelector.languageServerDefault": "({language}) ברירת מחדל",
   "components.LanguageSelector.originalLanguageDefault": "כל השפות",
+  "components.Layout.LanguagePicker.displaylanguage": "שפת התצוגה",
+  "components.Layout.SearchInput.searchPlaceholder": "חיפוש סרטים וסדרות",
+  "components.Layout.Sidebar.browsemovies": "סרטים",
+  "components.Layout.Sidebar.browsetv": "סדרות",
+  "components.Layout.Sidebar.dashboard": "Discover",
```
Comment (gauthier-th) [inline src/i18n/locale/he.json]: This one too, it is always visible in the sidebar

File: src/i18n/locale/he.json [hunk 7]
```diff
@@ -1,175 +1,1324 @@
...
+  "components.UserList.importfromJellyfin": "{mediaServerName} ייבוא משתמשי",
+  "components.UserList.importfromJellyfinerror": "Something went wrong while importing {mediaServerName} users.",
+  "components.UserList.importfrommediaserver": "{mediaServerName} ייבוא משתמשי",
+  "components.UserList.importfromplex": "Plex ייבוא משתמשי",
+  "components.UserList.importfromplexerror": "Something went wrong while importing Plex users.",
+  "components.UserList.localLoginDisabled": "The <strong>Enable Local Sign-In</strong> setting is currently disabled.",
+  "components.UserList.localuser": "משתמש מקומי",
+  "components.UserList.mediaServerUser": "{mediaServerName} משתמש",
+  "components.UserList.newJellyfinsigninenabled": "The <strong>Enable New {mediaServerName} Sign-In</strong> setting is currently enabled. {mediaServerName} users with library access do not need to be imported in order to sign in.",
+  "components.UserList.newplexsigninenabled": "The <strong>Enable New Plex Sign-In</strong> setting is currently enabled. Plex users with library access do not need to be imported in order to sign in.",
```
Comment (gauthier-th) [inline src/i18n/locale/he.json]: Here too

File: src/i18n/locale/he.json [hunk 8]
```diff
@@ -1,175 +1,1324 @@
...
+  "components.Settings.services": "שירותים",
+  "components.Settings.settingUpPlexDescription": ".יש ללחוץ על הכפתור לצד הרשימה כדי לטעון שרתים זמינים <RegisterPlexTVLink>plex.tv</RegisterPlexTVLink> ניתן לרשום את הפרטים באופן ידני או לבחור שרת מתוך ,Plex בכדי להגדיר את",
+  "components.Settings.sonarrsettings": "Sonarr הגדרות",
+  "components.Settings.ssl": "SSL",
+  "components.Settings.startscan": "התחל סריקה",
+  "components.Settings.syncJellyfin": "סריקת ספריות",
+  "components.Settings.syncing": "מסנכרן",
+  "components.Settings.tautulliApiKey": "API Key",
+  "components.Settings.tautulliSettings": "Tautulli הגדרות",
+  "components.Settings.tautulliSettingsDescription": "Optionally configure the settings for your Tautulli server. Jellyseerr fetches watch history data for your Plex media from Tautulli.",
```
Comment (gauthier-th) [inline src/i18n/locale/he.json]: Here

**Review by gauthier-th — CHANGES_REQUESTED** [review]
There is still a few missing translations. Could you please translate them?
I didn't go into details, I just highlighted the ones i saw.

**Comment by Fallenbagel** [comment]
> There is still a few missing translations. Could you please translate them? I didn't go into details, I just highlighted the ones i saw.

Oh yeah. I just went over it too quickly and didn't notice

**Review by Fallenbagel — CHANGES_REQUESTED** [review]
@mobihen please translate the missing ones

**Comment by mobihen** [comment]
@gauthier-th 
I've fixed missing translations as requested :)

Thanks

**Review by gauthier-th — APPROVED** [review]
LGTM

**Review by Fallenbagel — APPROVED** [review]

**Comment by mobihen** [comment]
@Fallenbagel 
Please merge while no conflicts :)
I do not have write access permissions to the repo.

**Comment by Fallenbagel** [comment]
@all-contributors please add @mobihen for translation

**Comment by controlfix87** [comment]
@fallenbagel 
Hi, Just wnated to know the reason that i'm not in the contributors sections.. 
Thanks :)

**Comment by gauthier-th** [comment]
@all-contributors please add @mobihen for translation

**Comment by gauthier-th** [comment]
> Hi, Just wnated to know the reason that i'm not in the contributors sections..

~We simply forgot :)~
~Oh sorry, yeah a merge issue happened in the readme, i'll make a PR to fix it~

**Comment by controlfix87** [comment]
Thank you very much!

**Comment by gauthier-th** [comment]
image

My bad I looked at it too fast - you're correctly listed as a contributor.

**Comment by controlfix87** [comment]
I did check before bud didnt see it.. well. all is OK. THANKS!

**Pull request merged**